tests/integration/test_sanity.py

```diff
@@ -189,6 +189,32 @@ def test_readonly_commands(cluster):
     assert cluster.node(1).client.get('key') == b'value'
 
 
+def test_readonly_acl_oom_permutations(cluster):
+    cluster.create(3)
+
+    cluster.execute('set', 'abc', 1234)
+    cluster.wait_for_unanimity()
+
+    # no oom, acl allow
+    assert cluster.execute("get", "abc") == b'1234'
+
+    # oom, acl allow
+    cluster.execute('config', 'set', 'maxmemory', 1)
+    assert cluster.execute("get", "abc") == b'1234'
+    cluster.execute('config', 'set', 'maxmemory', 0)
+
+    cluster.execute('acl', 'setuser', 'default', 'resetkeys')
+    # no oom, acl deny
```

Comment:
```suggestion
    # no oom, acl deny
    cluster.execute('acl', 'setuser', 'default', 'resetkeys')
```
Should we swap the lines?